tests/test_at.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+
+from contextlib import contextmanager, suppress
+from importlib import import_module
+from typing import TYPE_CHECKING
+
+import numpy as np
+import pytest
+from array_api_compat import (  # type: ignore[import-untyped]  # pyright: ignore[reportMissingTypeStubs]
+    array_namespace,
+    is_dask_array,
+    is_pydata_sparse_array,
+    is_writeable_array,
+)
+
+from array_api_extra import at
+
+if TYPE_CHECKING:
+    from array_api_extra._lib._typing import Array, Untyped
+
+all_libraries = (
```

Comment:
`all_libraries: Final` will help typecheckers understand that this is a constant, which they usually automatically assume if written in all-caps